src/sknnr/transformers/_base.py

```diff
@@ -11,3 +15,17 @@ def fit(self, X, y=None, sample_weight=None):
         scaler = super().fit(X, y, sample_weight)
         scaler.scale_ = np.std(np.asarray(X), axis=0, ddof=self.ddof)
         return scaler
+
+
+@contextmanager
+def set_temp_output(
+    transformer: TransformerMixin, temp_mode: Literal["default", "pandas"]
+):
+    """Temporarily set the output mode of an transformer."""
```

Comment:
OK, now I'm totally starting to doubt my English skills!  Did you mean "an transformer"?  Or was that left over from "an estimator"?

Reply:
Oops, that one's on me! Definitely a leftover